src/Table.luna

```diff
@@ -126,21 +283,107 @@ class Table:
         cols = self.toList
         columnValueLists = cols.each (_ . slice 0 rowCount . toList)
         JSON.empty . insert "header" self.columnNames . insert "data" columnValueLists
+    
+    # Returns `Table` value with records for which the given predicate returns `True`.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.filter (x: x.at "col2" < 14)
+    #
+    # `f`: The predicate (Boolean-valued function).
```

Comment:
I see two problems with "(Boolean-valued function).":
1) it looks like typename ersatz — we should either omit it or put the proper Luna type
2) strictly speaking this expression does not yield Bool anywhere

The most surprising thing (and needing explanation) for user is what is the type of "x" and what operations are allowed.